roadmap.md

```diff
@@ -1,24 +1,38 @@
 # Tekton Triggers Roadmap
 
-In 2019 we created a simple system for creating instances of Tekton resources, triggered
-by json payloads sent to HTTP endpoints (Tekton Triggers).
+In 2021, we'd like to get the Triggers APIs to a beta level of stability, add popular 
+feature requests, and plan for a v1 release.
 
-In 2020 we would like to add missing features and then push for a `beta` release in the
-same year.
+## H1 2021:
+* Get to Beta!
+    * [Trigger Templates - JSON escaping, validation, and templating](https://github.com/tektoncd/triggers/issues/697) 
+    * Trigger CRD - Path based Triggers
+    * v1beta1 version upgrade
+    * Beta docs/examples overhaul
 
-We are targeting improving the experience for both end users and operators:
+* Operator features:
+    * [RBAC/Permission setup for multi-tenant usage](https://github.com/tektoncd/triggers/issues/77)
+    * Customizable EventListeners
+        * Scale to Zero (Knative) EventListener
```

Comment:
We already have this support right :thinking: 

Is there any other usecase to keep this feature under Operator section ?

Reply:
This is the roadmap for things we want to do in 2021. Some of the features we have already added (such as the Knative EL), others we have not yet done. The roadmap documents all of them